src/Umbraco.Web/CompositionExtensions.cs

```diff
@@ -111,12 +117,7 @@ public static void SetDefaultRenderMvcController<TController>(this Composition c
         /// <remarks>The controller type is registered to the container by the composition.</remarks>
         public static void SetDefaultRenderMvcController(this Composition composition, Type controllerType)
         {
-            composition.OnCreatingFactory["Umbraco.Core.DefaultRenderMvcController"] = () =>
-            {
-                // no need to register: all IRenderMvcController are registered
-                //composition.Register(controllerType, Lifetime.Request);
-                Current.DefaultRenderMvcControllerType = controllerType;
-            };
+           // NOOP
```

Comment:
I guess at least we need a TODO here.. 
Not sure why this is so complicated and not just something like
 ```
composition.Services.AddScoped(typeof(IRenderMvcController), controllerType);
```

Reply:
I'm not sure I follow this, what's the TODO for?

This sets Current.DefaultRenderMvcControllerType but current wont exist.

Reply:
There's a TODO higher up the call stack to handle this, just removed the two extensions for now, they aren't called

Reply:
I know current won't exist in the long run, but currently, it is still used in "RenderRouteHandler", which is a class that soon we need to be migrated.

I guess in the migrated version, we will just inject the `IRenderMvcController` instead of using it from `Current.DefaultRenderMvcControllerType`, but right now we have some functionality that seems to work, but is just a noop-version.

No matter what, I don't see the idea of a noop-implementation? Either we can removed it (and replace with something similar to `composition.Services.AddScoped(typeof(IRenderMvcController), controllerType);`
otherwise I guess we at some point will need to implemented => Todo :)

Reply:
I see you removed the methods.. That makes more sense to me :)

Reply:
Oh this is a different "Current"

> composition.Services.AddScoped(typeof(IRenderMvcController), controllerType);

Wouldn't be anything like what it did though that code was commented out, all it did was set DefaultRenderMvcControllerType on Umbraco.Web.Composing.Current

Reply:
Enhanced the existing TODO in WebInitialComposer now also to give context later.